satellite-software/platformio.ini

```diff
@@ -9,7 +9,7 @@ lib_deps =
     https://github.com/matthijskooijman/arduino-lmic.git
 build_flags = 
     -D _lmic_config_h_
-    -D CFG_us915                ; change to CFG_eu868 in Europe
+    -D CFG_eu868
```

Comment:
I strongly suspect you are pushing you local build changes to this PR rather than the main branch in your forked repository. Changes like this I will not merge into my repository because as your commit title says, its your local build's config. 

Are you still intending for this PR to be merged into the main code?